assets/i18n/hi.json

```diff
@@ -17,7 +17,18 @@
 	"I18N_CREATE_YES_PLEASE": "हाँ!",
 	"I18N_DASHBOARD_COLLECTIONS": "कलेक्शन्स",
 	"I18N_DASHBOARD_EXPLORATIONS": "एक्सप्लोरेशन्स",
+	"I18N_DASHBOARD_EXPLORATIONS_SORT_BY": "सॉर्ट करें",
+    "I18N_DASHBOARD_EXPLORATIONS_SORT_BY_AVGRATING" : "औसत रेटिंग",
```

Comment:
could you use tabs instead of spaces just for this part, to make it fit the style of the rest of the file?